source/stylesheets/base/_mixins.scss

```diff
@@ -0,0 +1,22 @@
+@mixin generate-bracket($height) {
+  display: flex;
+  flex-direction: column;
+  height: $height;
+  justify-content: space-between;
+  margin-top: $base-spacing;
+  text-align: center;
+
+  *:first-child {
```

Comment:
You can delete lines 9–11. `Justify-content: flex-start;` is default so you don't need to declare anything.